Alicerce/Sources/Logging/Protocols/LogItemLevelFormatter.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  LogItemLevelFormatter.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+public protocol LogItemLevelFormatter {
+
+    var colorEscape: String { get }
+    var colorReset: String { get }
+
+    func colorString(for level: Log.Level) -> String
+    func labelString(for level: Log.Level) -> String
+}
+
+public extension LogItemLevelFormatter {
+
+    var colorEscape: String {
+        return ""
+    }
+
+    var colorReset: String {
+        return ""
+    }
+
+    func colorString(for level: Log.Level) -> String {
+
+        switch level {
+        case .verbose: return "📔  "
+        case .debug: return "📗  "
+        case .info: return "📘  "
+        case .warning: return "📙  "
```

Comment:
I would change this to the yellow book 📒 since it is more close to the ⚠️ sign's colour  😇 💄